### PR TITLE
[Done]Fault tolerant job

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/cloud/issues/213

### CODE REVIEW

File: go/paddlecloud/submit.go [hunk 1]
```diff
@@ -107,6 +112,7 @@ func NewSubmitter(cmd *SubmitCmd) *Submitter {
 func (s *Submitter) Submit(jobPackage string, jobName string) error {
 	// if jobPackage is not a local dir, skip uploading package.
 	_, pkgerr := os.Stat(jobPackage)
+	fmt.Println("1")
```
Comment (Yancey1989) [inline go/paddlecloud/submit.go]: How about use some useful comment instead of numbers.

File: paddlecloud/paddlejob/paddle_job.py [hunk 2]
```diff
@@ -101,11 +113,28 @@ def _get_pserver_container_ports(self):
             port += 1
         return ports
 
+    def _get_master_container_ports(self):
```
Comment (Yancey1989) [inline paddlecloud/paddlejob/paddle_job.py]: Maybe the master port will export the etcd port and master port?

File: go/paddlecloud/submit.go (hunk 1 above)
Comment (typhoonzero) [inline go/paddlecloud/submit.go]: Oh, that's test message, will remove.

File: paddlecloud/paddlejob/paddle_job.py (hunk 2 above)
Comment (typhoonzero) [inline paddlecloud/paddlejob/paddle_job.py]: Done.

**Pull request merged**